### PR TITLE
Use k8s-1.18 at kubevirtci

### PR DESCRIPTION
**Pull request by qinqon**
Signed-off-by: Quique Llorente <ellorent@redhat.com>

**Is this a BUG FIX or a FEATURE ?**:

/kind infrastructure

**What this PR does / why we need it**:
Since kubevirtci has the provider for it let's use it, it will help us
see if there issues at ocp-4.5 since it's based on 1.18.

**Special notes for your reviewer**:

**Release note**:

```release-note
Using k8s-1.18 at kubevirtci
```

### CODE REVIEW

File: test/e2e/handler/simple_bridge_and_bond_test.go [hunk 1]
```diff
@@ -147,7 +147,7 @@ var _ = Describe("NodeNetworkState", func() {
 				}
 				resetDesiredStateForNodes()
 			})
-			It("should have the linux bridge at currentState", func() {
+			FIt("should have the linux bridge at currentState", func() {
```
Comment (phoracek) [inline test/e2e/handler/simple_bridge_and_bond_test.go:150]: Bad!

File: test/e2e/handler/utils.go [hunk 2]
```diff
@@ -327,16 +327,18 @@ func getVLANFlagsEventually(node string, connection string, vlan int) AsyncAsser
...
 			// [1] https://bugs.centos.org/view.php?id=16533
 			output, err := cmd.Run("test/e2e/get-bridge-vlans-flags-el8.sh", false, node, connection, strconv.Itoa(vlan))
 			Expect(err).ToNot(HaveOccurred())
 			return strings.Split(string(output), " ")
 		} else {
+			By("Getting vlan filtering from json output")
 			parsedBridgeVlans := gjson.Parse(bridgeVlans)
 
-			vlanFlagsFilter := fmt.Sprintf("%s.#(vlan==%d).flags", connection, vlan)
+			vlanFlagsFilter := fmt.Sprintf("#(ifname==%s).vlans.#(vlan==%d).flags", connection, vlan)
```
Comment (phoracek) [inline test/e2e/handler/utils.go:341]: This change has nothing to do with k8s, it is related to the version of `bridge` on the host. Are we breaking our compatibility with the previous version by doing this? Can we keep the compatibility somehow?

Comment (qinqon) [reply test/e2e/handler/utils.go:341]: I want to check if we can get the same information with nmcli so we don't depend on this output.

Comment (qinqon) [reply test/e2e/handler/utils.go:341]: I think we can keep previous compatibility somehow just checking if we have `ifname` string, let me do it.

**Pull request merged**